influunt-app/app/plugins/roundSlider-1.3/roundslider.js

```diff
@@ -211,7 +211,15 @@
 
             this._bind(this.input, "blur", this._focusOut);
             this._bind(this.input, "change", this._focusOut);
-        },
+
+            this.input.keypress(function(event) {
+              var charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
+              var charStr = String.fromCharCode(charCode);
+              if (event.which != 8 && event.which != 13 && /\D/g.test(charStr)) {
```

Comment:
Tá usando aqui o `event.which` ao invés do `charCode` por algum motivo? E se o `event.which` for `undefined`?

E recomendo colocar as constantes 8 e 13 em variáveis com nomes explicativos. O 13 eu sei que é a tecla `enter`, mas o 8 não sei qual tecla é.